deploy/Datadog.Trace.ClrProfiler.WindowsInstaller/Datadog.Trace.ClrProfiler.WindowsInstaller.wixproj

```diff
@@ -10,15 +10,16 @@
     <ProjectGuid>3054de0e-f140-4758-b2da-0b9470c6ede1</ProjectGuid>
     <SchemaVersion>2.0</SchemaVersion>
     <OutputType>Package</OutputType>
-    <OutputPath>bin\$(Configuration)\$(Platform)\</OutputPath>
+    <!-- use a unique property to allow changing OutputPath
+         for this project without changing it for other projects -->
+    <MsiOutputPath Condition="'$(MsiOutputPath)' == ''">bin\$(Configuration)\$(Platform)\</MsiOutputPath>
+    <OutputPath>$(MsiOutputPath)</OutputPath>
     <IntermediateOutputPath>obj\$(Configuration)\$(Platform)\</IntermediateOutputPath>
     <SuppressPdbOutput>True</SuppressPdbOutput>
     <DefineSolutionProperties>false</DefineSolutionProperties>
     <OutputName>datadog-dotnet-apm-1.16.1-$(Platform)</OutputName>
-  </PropertyGroup>
-
-  <PropertyGroup>
-    <DefineConstants>InstallerVersion=1.16.1</DefineConstants>
+    <TracerHomeDirectory Condition="'$(TracerHomeDirectory)' == ''">$(MSBuildThisFileDirectory)..\..\src\bin\tracer-home</TracerHomeDirectory>
+    <DefineConstants>InstallerVersion=1.16.1&#59;TracerHomeDirectory=$(TracerHomeDirectory)</DefineConstants>
```

Comment:
What is the `&#59;` sequence? Would a simple `;` suffice?

Reply:
The semicolon has special meaning (list separator) for msbuild, so it needs to be escaped here. Although, apparently the correct way to do this is `%3B;`, not `&#59;`. I'll test that and update.
https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-special-characters

Reply:
I tested with a simple semicolon and that seems to work. I couldn't tell you what the issue I had previously was since that was a few months ago and I don't remember. Changed in d825b365.